tests/test.js

```diff
@@ -5,16 +5,17 @@
 /*eslint-env node*/
 const request = require('supertest');
 const app =  require('../app.js');
+var cookie;
 
 describe('POST /items', function() {
```

Comment:
split your tests to multiple modules, based on what you are testing.
login, logout, protected/unprotected routes -> one test module
CRUD operations on items -> another module
https://github.com/ghsukumar/SFDC_Best_Practices/wiki/F.I.R.S.T-Principles-of-Unit-Testing

Reply:
tests have been move into login.test.js and CRUD operations are in items.test.js file. I should probably rename those. Due to creating more tests files I was forced to move app.listen into newly created server.js file.
Tests are passing, butI still need to log at the start of the file to pass cookie.